app/src/main/java/com/iron/espresso/presentation/home/study/list/StudyListActivity.kt

```diff
@@ -0,0 +1,93 @@
+import androidx.fragment.app.Fragment
+import com.google.android.material.tabs.TabLayout
+import com.iron.espresso.R
+import com.iron.espresso.base.BaseActivity
+import com.iron.espresso.databinding.ActivityStudyListBinding
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class StudyListActivity :
+    BaseActivity<ActivityStudyListBinding>(R.layout.activity_study_list) {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        setToolbarTitle(getString(R.string.study))
+        setNavigationIcon(R.drawable.ic_back_24)
+
+        val fragmentManager = supportFragmentManager
+
+        fragmentManager.beginTransaction()
+            .replace(
+                R.id.study_list_container,
+                NewListFragment.newInstance(),
+                resources.getString(R.string.recency)
+            ).commit()
+
+        val studyTabList = resources.getStringArray(R.array.study_tab)
+        studyTabList.forEach { title ->
+            binding.topTab.addTab(binding.topTab.newTab().setText(title))
+        }
+
+        binding.topTab.addOnTabSelectedListener(object : TabLayout.OnTabSelectedListener {
+            override fun onTabSelected(tab: TabLayout.Tab?) {
+
+                val addFragment = getFragment(tab?.position ?: 0)
+
+                val hideFragment = fragmentManager.fragments.find {
+                    if (tab?.position == 0) {
+                        it is LocationFragment
+                    } else {
+                        it is NewListFragment
+                    }
+                }
+                if (hideFragment != null) {
+                    fragmentManager.beginTransaction()
+                        .hide(hideFragment).commit()
+                }
+
+                val findFragment = fragmentManager.findFragmentByTag("${tab?.text}")
+                if (findFragment == null) {
+                    fragmentManager.beginTransaction()
+                        .add(R.id.study_list_container, addFragment, "${tab?.text}").commit()
+                } else {
+                    fragmentManager.beginTransaction().show(findFragment).commit()
+                }
+            }
+
+            override fun onTabUnselected(tab: TabLayout.Tab?) {
+            }
+
+            override fun onTabReselected(tab: TabLayout.Tab?) {
+            }
+        })
+    }
+
+    private fun getFragment(position: Int): Fragment {
+        return when (position) {
+            0 -> NewListFragment.newInstance()
+            1 -> LocationFragment.newInstance()
+            else -> error("")
+        }
+    }
+
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        when (item.itemId) {
+            android.R.id.home -> {
+                onBackPressed()
+            }
+        }
+        return true
+    }
+
+    companion object {
+        fun getInstance(context: Context) =
+            Intent(context, StudyListActivity::class.java)
+    }
```

Comment:
```suggestion
        fun getIntent(context: Context) =
            Intent(context, StudyListActivity::class.java)
    }
```
뭔가 instance를 가져온다기 보다는, Activity 띄우는데 사용할 Intent객체를 가져오는 것이니 요런 네이밍이 어떨까 해요

Reply:
List부분 수정 완료했습니다